.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+    "singleQuote": true,
```

Comment:
Ben hier persoonlijke geen fan van, zie liever dat deze setting op false blijft. Maar als dit een gemeente standaard is dan volg ik die natuurlijk.

Reply:
in alle andere projecten is het wel single quote: mij maakt het niet uit kies jij maar